application/dash_application/views/pre_exome.py

```diff
@@ -18,6 +18,7 @@
     # Buttons
     'update-button',
     'download-button',
+    'all',
```

Comment:
Perhaps `all-runs` instead of just `all`?

Reply:
done